pom.xml

```diff
@@ -914,6 +914,7 @@ under the License.
     <module>maven-skins</module>
     <module>doxia-tools</module>
     <module>docs</module>
+    <module>maven-shared-doc-resource</module>
```

Comment:
I will try to work on MSITE-911 in the near future, so this doesn't need to be maintained outside the parent project.

Reply:
My approach to put it in external repo are:
- we can use it in ASF parent
- maybe other ASF project try to use it - eg commons use own template generated by deprecated Maven/Ant plugin https://github.com/apache/commons-build-plugin - @garydgregory WDYT

Reply:
Hello @slawekjaranowski 

The Commons Build and Release plugins are not deprecated and generates a lot more than just one page. I use them for all Commons releases.

What I'm more interested in getting rid of is the commons-skin plug-in but I don't know if it's possible to fold it in commons-parent directly.

It would be nice to merge the Build and Release plugin but I never took the time to do it. 

Ideally, I'd prefer to have all three current plug-ins in commons-parent, so I have 1 framework-y thing to deal with instead of 4.

Reply:
I know that  Build and Release plugins are not deprecated but use deprecated ant mojo